designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,263 @@
+- Repo: No ESLint repos, only the ones mentioned below (owned by private
+  persons)
+- Start Date: 2022-05-31
+- RFC PR:
+- Authors: [@aladdin-add](https://github.com/aladdin-add),
+  [@MichaelDeBoey](https://github.com/MichaelDeBoey),
+  [@ota-meshi](https://github.com/ota-meshi) &
+  [@voxpelli](https://github.com/voxpelli)
+
+# `@eslint-community` GitHub organisation
+
+## Summary
+
+<!-- One-paragraph explanation of the feature. -->
+
+Facilitate a GitHub organisation
+([@eslint-community](https://github.com/eslint-community)) where community
+members could maintain widely used ESLint related packages that would otherwise
+get abandoned.
+
+## Motivation
+
+<!-- Why are we doing this? What use cases does it support? What is the expected
+outcome? -->
+
+There a some widely used ESLint related packages that aren't actively maintained
+anymore and we'd like to transfer them to a community owned GitHub organization
+(`@eslint-community`), so we can give them back to the community.
+
+## Detailed Design
+
+<!--
+   This is the bulk of the RFC.
+
+   Explain the design with enough detail that someone familiar with ESLint
+   can implement it by reading this document. Please get into specifics
+   of your approach, corner cases, and examples of how the change will be
+   used. Be sure to define any new terms in this section.
+-->
+
+### How do you envision `@eslint-community`?
+
+Just like the [@jest-community](https://github.com/jest-community) org, the
+`@eslint-community` org would be a collection of important/widely used ESLint
+plugins/packages that aren't maintained anymore and are given back to the
+community.
+
+> Community repos for ESLint related projects
+
+The projects could be released under the `@eslint-community` scope in a later
+stage.
```

Comment:
Are you referring to npm scope? I think the npm scope is even more important than the GitHub org, as getting access to publish on npm is the one critical part in adding new maintainers.

On that note: I would propose enforcing a fully automated release flow through something like [`google-github-actions/release-please-action`](https://github.com/google-github-actions/release-please-action) as that makes it so that individual maintainers does not need to be added to npm, instead their access to merge PR:s is enough.

Reply:
another important thing: the github/npm members must enable 2FA.

Reply:
> Are you referring to npm scope? I think the npm scope is even more important than the GitHub org, as getting access to publish on npm is the one critical part in adding new maintainers.

I was indeed referring to the `@eslint-community` npm scope, which I currently created already to make sure we don't loose it.

CC/ @nzakas I'm happy to transfer ownership to the ESLint team if that's preferred.

> I would propose enforcing a fully automated release flow through something like [`google-github-actions/release-please-action`](https://github.com/google-github-actions/release-please-action) as that makes it so that individual maintainers does not need to be added to npm, instead their access to merge PR:s is enough.

@kentcdodds set  up something similar for @remark-embedder & @testing-library orgs, where a bot account is actually publishing new versions & people only need write access on the repo. CI + the bot would do the rest.

> the github/npm members must enable 2FA

I think that would indeed be a good idea 👍

Reply:
If the `@eslint-community` org would never fork projects but only accept projects that are moved into it by their original maintainers, then the npm scope is much less important. So this reconnects to the https://github.com/eslint/rfcs/pull/91#discussion_r885800643 discussion.

Reply:
I think the goal should be to transfer the original repository to the new org.

If that's not possible because we can't get in touch with the owner or due to other reasons, I think a fork would unblock the community.

Although I think that if we would have a fork and we could transfer the original repo to the org at a later point in time, we should do so & see that we bring the original repo up to date with the fork first before moving forward.